cumulusci/tasks/salesforce/UpdateDependencies.py

```diff
@@ -21,6 +20,9 @@ class UpdateDependencies(BaseSalesforceMetadataApiTask):
     api_class = ApiDeploy
     name = "UpdateDependencies"
     task_options = {
+        "dependencies": {
+            "description": "List of dependencies to update. Defaults to project__dependencies"
```

Comment:
in what form?

Reply:
Same form as `project__dependencies` (a list of dicts specifying either a github url or namespace). I wasn't sure how to explain it clearly without giving an example, which there isn't really room for here.

Reply:
Better now?